codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -239,6 +239,7 @@ void removeStream(DefaultStream stream) {
         private State state = IDLE;
         private short weight = DEFAULT_PRIORITY_WEIGHT;
         private DefaultStream parent;
+        // TODO(nathanmittler): We no longer need a map. LinkedList?
```

Comment:
Humm.  If the children count is low a the `LinkedList` should be fine for the `put` and `remove` type operations...however as the count grows this will not surprisingly drop off.  I would say either:
1. Leave it as a map.
2. Provide an interface to be able create a new "collection" and returns a "map" like interface that we need to use.  The user can swap out the collection depending on what make sense for their use case...?